BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+cc_library(
+    name = "cpptrace",
+    srcs = glob([
+        "src/**/*.hpp",
+        "src/**/*.cpp",
+    ]),
+    local_defines = [
+        "CPPTRACE_GET_SYMBOLS_WITH_LIBDWARF",
+        "CPPTRACE_DEMANGLE_WITH_CXXABI",
+        "CPPTRACE_UNWIND_WITH_UNWIND",
```

Comment:
Circling back to this, happy to go with this for now but it'd be nice to let users choose other back-ends if they want (e.g. libunwind). How hard would that be to support?

Reply:
Letting the user choose is always the best approach. The most challenging part is converting other backends to be built with Bazel. Fortunately, Bazel's `foreign_cc` rule can help us. I'll try to make other backends work

Reply:
Thank you so much, I really appreciate all the work! If it proves nontrivial, libunwind is the main back-end that would be useful to support for users as it's the only way to do signal-safe unwinding.

Reply:
You're welcome! Could you also enable this PR to automatically trigger the GitHub workflows? That way, I can test it myself.

Reply:
I don't see a way to not require me to hit approve, but I can hit approve